src/refactorings/split-declaration-and-initialization/split-declaration-and-initialization.ts

```diff
@@ -32,9 +32,12 @@ function updateCode(ast: t.AST, selection: Selection): t.Transformed {
         ),
         ...declarations
           .filter(isDeclarationInitialized)
-          .map(({ id, init }) =>
-            t.expressionStatement(t.assignmentExpression("=", id, init))
-          )
+          .map(function ({ id, init }) {
+            if (id.type == "Identifier" && "typeAnnotation" in id) {
```

Comment:
I think we can write:

```ts
if (t.isIdentifier(id)) {
```

And that should work. It will be less dependent of the implementation details.

Reply:
It looks like you merged into master without adding your suggestion. Did you change your mind?

Reply:
@iulspop nope, I did it faster. It wasn't blocking so simpler to get this merged then refactor on master 👍 

https://github.com/nicoespeon/abracadabra/commit/9b6624b1e682ffbcbf9cd43ca36e43cfbf275059